### DISCUSSION TITLE
FVMBAG1 runs to NaN in 0001.out

### DISCUSSION THREAD

**Discussion by Heguo-Ou** [OpenRadioss/OpenRadioss#3081] [Q&A]
Are there anyone encounter the issue "FVBAG control the time step and ts drop down to ~0, then model fly away with NaN info in 0001.out"?  
What is the most likely cause of this? More，the particularly weird is that after changing the vent hole area scale factor 'Avent' , the FVMBAG1 model runs normally to end.

I don't understand what this has impact with the vent hole Avent data field....

**Comment by Heguo-Ou** [comment]
![20241019223545](https://github.com/user-attachments/assets/69f55095-c34f-47a8-b67c-57a16070e143)

**Comment by Schwoertzig** [comment]
Hi @Heguo-Ou ,
To properly address this question, a debug analysis would be necessary. If you can share a reproducer, I could help to take a look.
Regards

**Comment by Heguo-Ou** [comment]
Change to issue and more finding in this link:
https://github.com/OpenRadioss/OpenRadioss/issues/3409

**Comment by MarianBulla** [comment]
Hi @Heguo-Ou,
thanks for pointing it to the link.

Yes, FVM-Airbags are not supported in OpenRadioss, due to some technical reasons.

Thanks and best regards,
Marian
